ci-operator/step-registry/openshift-docs/openshift-docs-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: openshift-docs
+  from: base
+  commands: openshift-docs-commands.sh
+  credentials:
+  - mount_path: /var/run/openshift-docs-netlify-secret
+    name: netlify-secret
+    namespace: test-credentials
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
```

Comment:
Need to set
`env:` for the netlify secret.

Reply:
Or not use the ref at all. https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/#use-a-secret-in-non-step-jobs